llvm/include/llvm/IR/ReplaceConstant.h

```diff
@@ -18,10 +18,13 @@ namespace llvm {
 
 template <typename T> class ArrayRef;
 class Constant;
+class Function;
 
 /// Replace constant expressions users of the given constants with
 /// instructions. Return whether anything was changed.
-bool convertUsersOfConstantsToInstructions(ArrayRef<Constant *> Consts);
+bool convertUsersOfConstantsToInstructions(ArrayRef<Constant *> Consts,
```

Comment:
Please add descriptions of the new parameters to the function in the comment above.